### PR TITLE
added tests for quantized tflite sin operator

### PR DESCRIPTION
**Pull request by onkar-sima-ai**
https://github.com/apache/tvm/issues/9187

I have added support for quantized SIN tflite operator and wrote tests for the same.

Thanks for contributing to TVM!   Please refer to guideline https://tvm.apache.org/docs/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

### CODE REVIEW

File: tests/python/frontend/tflite/test_forward.py [hunk 1]
```diff
@@ -1850,6 +1878,16 @@ def _test_tan(data):
...
+#######################################################################
+# Sqrt
+# ----
+
+
+def _test_sqrt(data):
+    """One iteration of sqrt"""
+    return _test_unary_elemwise(math_ops.sqrt, data)
+
+
```
Comment (leandron) [inline tests/python/frontend/tflite/test_forward.py:1890]: This block above seems unrelated to this PR, right?

**Review by leandron — CHANGES_REQUESTED** [review]

File: tests/python/frontend/tflite/test_forward.py (hunk 1 above)
Comment (onkar-sima-ai) [reply tests/python/frontend/tflite/test_forward.py:1890]: Yes I will remove them.

**Review by leandron — COMMENTED** [review]
The PR in general LGTM.

I see there are a few linting errors when running this in CI. You can see on this page how to reproduce linting errors locally on your machine: https://tvm.apache.org/docs/contribute/pull_request.html

You can see the errors here as well: https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-9478/2/pipeline

Once we get this PR past CI, I'm happy for this to be merged.

**Comment by onkar-sima-ai** [comment]
Hi @leandron I am not sure the failure of CI has to do anything with the changes I introduced.
Is this an expected condition?
Can you please help me understand why CI failed?

**Comment by onkar-sima-ai** [comment]
Hi @leandron 
I ran the failing test in my local, and it passed. 
I was not able to reproduce the same error.
So I rebased and pushed again.
Can you please let me know if we should wait to see if CI passes or Is there something wrong with my PR?

**Comment by onkar-sima-ai** [comment]
Hi @leandron  The CI has passed, Can you please merge this PR https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-9478/4/pipeline/269/

**Review by leandron — APPROVED** [review]
LGTM, thanks @onkar-sima-ai !
